main/kotlin/com/example/people_managment_system/service/SystemService.kt

```diff
@@ -0,0 +1,177 @@
+package com.example.people_managment_system.service
+
+import com.example.people_managment_system.dtos.EmployeeDto
+import com.example.people_managment_system.model.Employee
+import com.example.people_managment_system.repositories.CompanyRepository
+import com.example.people_managment_system.repositories.EmployeeRepository
+import com.example.people_managment_system.repositories.ManagerRepository
+import com.example.people_managment_system.repositories.TeamRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.dao.DataIntegrityViolationException
+import org.springframework.stereotype.Service
+
+@Service
+
+class SystemService(

```

Comment:
When you commend code, you should always look to comment public classes and methods.

You can use KDoc for this.

It looks like this:

/**
* ...
* ...
**